readme.md

```diff
@@ -294,6 +294,7 @@
 - [gotql](https://github.com/khaosdoctor/gotql) - GraphQL request library built on [got](https://github.com/sindresorhus/got).
 - [global-agent](https://github.com/gajus/global-agent) - Global HTTP/HTTPS proxy agent that is configurable using environment variables.
 - [smoke](https://github.com/sinedied/smoke) - File-based HTTP mock server with recording abilities.
+- [undici](https://github.com/nodejs/undici) - HTTP/1.1 client written from scratch.
```

Comment:
This is not a good description. It should describe what it as *and* the benefits of this package.

Reply:
Thanks. I've update with someting more descriptive.